src/Curl.CommandLine.Parser/Models/Parsing/ParameterEvaluator.cs

```diff
@@ -19,7 +19,7 @@ public bool CanEvaluate(string handle)
 
     public void Evaluate(ref Span<char> commandLine, ConvertResult<CurlOptions> convertResult)
     {
-        commandLine = commandLine.TrimStart(Chars.Escape).Trim();
+        commandLine = commandLine.ToString().TrimStart(Chars.Escape).Trim().ToCharArray();
```

Comment:
Why do we need these changes?

Reply:
This is because of the [Trim method](https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.trim?view=net-6.0#system-memoryextensions-trim-1(system-span((-0))-system-readonlyspan((-0)))) which is not available in netstandard2.1

Reply:
@jiahao1553 ouch, let's drop .netstandard2.1 then. Let's target .net core 3.0+

Reply:
Done updating.